### PR TITLE
add autofixer

### PR DESCRIPTION
**Pull request by wzrdtales**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,54 +1,80 @@
...
+"use strict";
 
-module.exports = function (context) {
-  var allowOneLiners = context.options[0] === 'allow-one-liners';
-  var maxInOneLiner = context.options[1] !== undefined ? context.options[1] : 1;
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        "require `let` or `const` instead of `var` when outside of try...catch",
```
Comment (cjihrig) [inline lib/index.js]: This rule doesn't do that.

File: lib/index.js [hunk 2]
```diff
@@ -1,54 +1,80 @@
...
+    return {
+      ArrowFunctionExpression: checkArrow,
+      FunctionExpression: checkFunction,
+      FunctionDeclaration: checkFunction
+    };
+  }
 };
 
-module.exports.esLintRuleName = 'hapi-scope-start';
+module.exports.esLintRuleName = "hapi-scope-start";
```
Comment (cjihrig) [inline lib/index.js]: Please undo any stylistic changes. They are causing the linter to fail (`npm test`).

File: lib/index.js (hunk 1 above)
Comment (wzrdtales) [reply lib/index.js]: Yeh sry, just copy pasted it from another example.

File: lib/index.js (hunk 2 above)
Comment (wzrdtales) [reply lib/index.js]: yeah will do, my linter + fixer automatically screws up any project that has no proper .eslint config in their repository :/

**Comment by wzrdtales** [comment]
Just dropping this here maybe investing some time in it at a later time, I just pushed these additions back to your repo since they were missing and it was annoying to work with hapi packages that rely on that rule. Hope it will help you anyway.

I fixed your remarks though.

**Comment by hueniverse** [comment]
Closing due to lack of progress.

**Pull request closed without merging**